roles/contiv_network/tasks/ovs_cleanup.yml

```diff
@@ -28,6 +28,7 @@
 - name: deny openvswitch_t type in selinux
   shell: >
       semanage permissive -d openvswitch_t
+  when: (ansible_os_family == "RedHat")
```

Comment:
@mapuri : why ? It will not run on ubuntu , resulting that task will skipped as opposed to the current logic : It will still run even if its ubuntu and fail.